packages/MarkdownEditor-Tests.package/MarkdownInlineParserTest.class/instance/testDontFindOpenerForEmphasisDelimiter.st

```diff
@@ -0,0 +1,17 @@
+tests - tokenization
+testDontFindOpenerForEmphasisDelimiter
```

Comment:
The situation would be better described as `testCloserDelimiterWithoutOpener` or similiar